CMakeLists.txt

```diff
@@ -729,8 +729,12 @@ else()
     set(SUMMARY_LIBRARY_TYPE "Shared")
     add_library(LLGL SHARED ${FilesLLGL})
 endif()
+set(all_targets LLGL)
 
-target_include_directories(LLGL PUBLIC ${PROJECT_INCLUDE_DIR})
+include(GNUInstallDirs)
```

Comment:
In the vcpkg patch you linked I only see a few additional `install` commands, but where does this `GNUInstallDirs` for instance come from?

Reply:
It comes from the part `+ make it work with find_package`

Reply:
No it it not only required for vcpkg, it is required for the variable `CMAKE_INSTALL_INCLUDEDIR` defined by this module. The module is also required always (it is not specific to GNU/Linux, their naming was maybe bad. I guess the name comes from `Provides install directory variables as defined by the GNU Coding Standards.` )